Knoema.Client/Search/TimeseriesSearch/TimeseriesSearchFacets.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+
+namespace Knoema.Search.TimeseriesSearch
+{
+	public class TimeseriesSearchFacets
+	{
+		public TimeseriesSearchDataSourceInfos DataSourceInfos { get; set; }
```

Comment:
special class is redundant, it can be just Dictionary<string, DatasetSourceItemViewModel>

Reply:
done